README.md

```diff
@@ -1,13 +1,12 @@
 # Survey form with React
```

Comment:
Hi Karan! I will do the PR of this week's project. If you have any questions, feel free to reach out to me directly.